src/App.tsx

```diff
@@ -1,14 +1,23 @@
+import { Route, Routes } from 'react-router-dom';
+
 import styles from '@/App.module.css';
 import About from '@/components/About/About.tsx';
 import FooterComponent from '@/components/Footer/Footer.component.tsx';
 import Header from '@/components/Header/Header.tsx';
+import { ProductsList } from '@/components/ProductsList/ProductsList.tsx';
 
 function App() {
+    const URL_ROUTE = '/fe-react-2024';
+
     return (
         <>
             <div className={styles.wrapper}>
-                <Header />
-                <About />
+                <Header route={URL_ROUTE} />
+                <Routes>
```

Comment:
It's a good idea to study react routing, and it will be better to start with conditional rendering 

![image](https://github.com/DavidTereshchenko/fe-react-2024/assets/108006836/384a34b1-e9ca-4021-ad5f-9c2da0f34646)

If you want to continue with routing, you need to accomplish it perfect functionality. Github has some troubles with routing, you will need to figure out how deploy you app

That is why I'm highly recommend you to implement conditional rendering